src/main/java/devs/toz/todolistmicroservice/model/Task.java

```diff
@@ -0,0 +1,20 @@
+package devs.toz.todolistmicroservice.model;
+
+import jakarta.persistence.*;
+import lombok.*;
```

Comment:
Don't use the wildcard here. Import only what you need. Using the wildcard would import everything from persistence and lombok and it would result in unnecessary imports.